net-dns/avahi/avahi-0.7-r1.ebuild

```diff
@@ -186,7 +186,8 @@ multilib_src_install() {
 	use mdnsresponder-compat && dosym avahi-compat-libdns_sd/dns_sd.h /usr/include/dns_sd.h
 
 	if multilib_is_native_abi && use doc; then
-		dohtml -r doxygen/html/. || die
+		docinto html
+		dodoc -r doxygen/html/.
```

Comment:
use HTML_DOCS instead.

Reply:
well, my intent was just to keep it simple and replace deprecated dohtml call. just this, not completely rewriting how docs are installed.

using HTML_DOCS will require calling einstalldocs.
It can be tricky because of how the ebuild is composed.
currently it conditionally  installs html docs in one of multilib phases.
and later installs text docs in all* phase.
I would like to leave it up to maintainer and just replace dohtml, that's it.

Reply:
i'm okay with the dodoc.